.eslintignore

```diff
@@ -0,0 +1 @@
+tests/dummy/public/large-file-skipped.js
```

Comment:
?

Reply:
JS files over some limit are ignored by ember-percy when discovering assets, so that file exists to test that. It was confusing the linter however.